docs/advanced.rst

```diff
@@ -739,277 +741,393 @@ Thus, for a higher performance, it can be appropriate to configure the reliabili
+announcements will bloat the network with meta traffic, but too scarce ones will delay the discovery of late joiners.
+
+Participant's announcement period is specified as a time expressed in seconds and nanosecond using a ``Duration_t``.
+Its default value is 3 seconds.
+
++---------------------------------------------------------+
+| **C++**                                                 |
++---------------------------------------------------------+
+| .. literalinclude:: ../code/CodeTester.cpp              |
+|    :language: c++                                       |
+|    :start-after: //CONF-DISCOVERY-LEASE-ANNOUNCEMENT    |
+|    :end-before: //!--                                   |
++---------------------------------------------------------+
+| **XML**                                                 |
++---------------------------------------------------------+
+| .. literalinclude:: ../code/XMLTester.xml               |
+|    :language: xml                                       |
+|    :start-after: <!-->CONF-DISCOVERY-LEASE-ANNOUNCEMENT |
+|    :end-before: <!--><-->                               |
++---------------------------------------------------------+
+
+.. _simple_disc_settings:
+
+SIMPLE Discovery Settings
+=========================
+
+The SIMPLE discovery protocol resolves the establishment of the end-to-end connection between various RTPS entities
+communicating via the RTPS protocol.
+Fast-RTPS implements the SIMPLE discovery protocol to provide compatibility with the
+`RTPS standard <https://www.omg.org/spec/DDSI-RTPS/2.2/PDF>`_.
+The specification splits up the SIMPLE discovery protocol into two independent protocols:
+
+- **Simple Participant Discovery Protocol (SPDP):** specifies how Participants discover each other in the network; it
+  announces and detects the presence of participants in a domain.
+
+- **Simple Endpoint Discovery Protocol (SEDP):** defines the protocol adopted by the discovered participants for the
+  exchange of information in order to discover the RTPS entities contained in each of them, i.e. the writers and
+  readers Endpoints.
+
++--------------------------+-----------------------------------------------------------------------+
+| Name                     | Description                                                           |
++==========================+=======================================================================+
+| `Initial Announcements`_ | It defines the behavior of the RTPSParticipant initial announcements. |
++--------------------------+-----------------------------------------------------------------------+
+| `Simple EDP Attributes`_ | It defines the use of the SIMPLE protocol as a discovery protocol.    |
++--------------------------+-----------------------------------------------------------------------+
+
+
+.. _`Initial Announcements`:
+
+Initial Announcements
```

Comment:
True enough

Reply:
How about:

> [RTPS standard](https://www.omg.org/spec/DDSI-RTPS/2.2/PDF) simple discovery mechanism requires the participant to send constant announcements. This announcements are not deliver in a reliable fashion and can be dispose of by the network. In order to avoid the discovery delay induced by message disposal, the initial announcement can be set up to make several shots to increase proper reception chances.

Reply:
Ok, but corrected:
> [RTPS standard](https://www.omg.org/spec/DDSI-RTPS/2.2/PDF) simple discovery mechanism requires the participant to periodically send announcements.
These announcements are not delivered in a reliable fashion, and can be disposed of by the network.
In order to avoid the discovery delay induced by message disposal, the initial announcement can be set up to make several shots, in order to increase proper reception chances.

Reply:
And I would also add a sentence explaining that, after all the initial announcements have been sent, the announcement period is used.

Reply:
done